readme.md

```diff
@@ -1,94 +1,115 @@
+
+Example Express endpoint to accept webmentions:
+
 ```typescript
-//Regular express setup
+// Regular Express setup
 import express from 'express';
 import bodyParser from 'body-parser';
 import webMentionHandler from 'path/to/your/webMentionHandler/file.ts';
 
 const app = express();
-// Web Mentions require the ability to parse Form encoded data from a post request.
+
+// Webmentions require the ability to parse Form encoded data from a post request.
 app.use(bodyParser.urlencoded({ extended: true }));
 
 app.post('/webmentions', async function(req, res) {
   try {
     // Pass the source and target of the request to the handler.
     // No need to run validation on it as this will be validated by the handler.
-    const recomendedResponse = await webMentionHandler.addPendingMention(req.body.source, req.body.target);
+    const recommendedResponse = await webMentionHandler.addPendingMention(req.body.source, req.body.target);
+
     // The response code is dictated by the specification but not enforced by the handler
-    // in case you need to action your own out-of-spec actions. response body should be human-readable
-    res.status(recomendResponse.code).send('accepted');
-  } catch (e) {
-    res.status(400).send(e.message);
+    // in case you need to action your own out-of-spec actions. The response body should be human-readable
+    res.status(recommendedResponse.code).send('accepted');
+  } catch (error) {
+    res.status(400).send(error.message);
   }
 });
 
-// open your express server
+// Open your Express server
 app.listen(8080);
 ```
+
 Once your endpoint is set up, you can set up a regular job with the following code to convert pending mentionNotifications into mention objects in your database that you can use.
+
 ```typescript
 import webMentionHandler from 'path/to/your/webMentionHandler/file.ts';
 
 setInterval(() => webMentionHandler.processPendingMentions(), 300000);
 ```
-I'd highly recommend processing mentions in an ephemeral cloud function, rather than a setInterval on your main node application, so that you don't accidentally leak the IP of your server to third parties. Especially if you aren't using the `whitelist` option for source hosts.
+
+I'd highly recommend processing mentions in an ephemeral cloud function, rather than a `setInterval` on your main Node application, so that you don't accidentally leak the IP of your server to third parties. Especially if you aren't using the `whitelist` option for source hosts.
 
 You can now grab the type of mention that you need for any given page.
+
 ```typescript
 import webMentionHandler from 'path/to/your/webMentionHandler/file.ts';
 
-const likes = webMentionHandler.getMentionsForPage('/blog/example-post', 'likes');
-const comments = webMentionHandler.getMentionsForPage('/blog/example-post', 'commments');
+const likes = webMentionHandler.getMentionsForPage('/blog/example-post', 'like');
+const replies = webMentionHandler.getMentionsForPage('/blog/example-post', 'reply');
```

Comment:
I was going to say this was incorrect, but looking at the code I use on https://mikevdv.dev. Seems that I updated this query at some point and didn't update the docs